### PR TITLE
Wrap Repay and Mint Vai with Enable Token Component

### PR DESCRIPTION
**Pull request by coreyar**
The PR wrap the the Repay and Mint VAI component in the Enable Token Component so that the user will only see the form if they've enable the token.

If they haven't enable the token they will be prompted to enable it

### CODE REVIEW

File: src/pages/Dashboard/MintRepayVai/MintVai/index.spec.tsx [hunk 1]
```diff
@@ -108,5 +118,32 @@ describe('pages/Dashboard/MintRepayVai/MintVai', () => {
     });
   });
 
+  it('asks the user to enable token if not enabled', async () => {
```
Comment (therealemjy) [inline src/pages/Dashboard/MintRepayVai/MintVai/index.spec.tsx]: I saw you added this test for the supply/withdraw modal too. I think we should add tests to the `EnableToken` component instead, so that we don't have to duplicate them on every component that uses it.

File: src/pages/Dashboard/MintRepayVai/RepayVai/index.spec.tsx [hunk 2]
```diff
@@ -101,5 +112,32 @@ describe('pages/Dashboard/MintRepayVai/RepayVai', () => {
     });
   });
 
+  it('asks the user to enable token if not enabled', async () => {
```
Comment (therealemjy) [inline src/pages/Dashboard/MintRepayVai/RepayVai/index.spec.tsx]: Same thing here about testing `EnableToken` instead

**Comment by therealemjy** [comment]
I found two issues with this:
- The XVS icon doesn't seem to display in the `EnableToken` component
Screen Shot 2022-05-03 at 17 37 40
- The `EnableToken` component displays when the user is logged out, however in the designs it's the interface with the form and a button indicating "Connect wallet" that should display. To be honest I think that's an inconsistent UX anyway, so I'd propose instead we do like we do for the borrow/supply modals and use the `ConnectWallet` component to handle this case. What do you think?

**Review by therealemjy — APPROVED** [review]
